game_eggs/urbanterror/egg-urban-terror4-3-4.json

```diff
@@ -0,0 +1,152 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2024-02-09T19:34:16+00:00",
+    "name": "Urban Terror 4.3.4",
+    "author": "Juli199696@web.de",
+    "description": "Urban Terror started as a realism based \"total conversion\" mod for Quake III Arena. Nowadays, Urban Terror is played using the free, stand-alone, ioquake3 engine modified, which doesn't require Quake III Arena, in addition to being an add-on to id Software's Quake III Arena.",
+    "features": null,
+    "docker_images": {
+        "quay.io\/pterodactyl\/core:glibc": "quay.io\/pterodactyl\/core:glibc"
+    },
+    "file_denylist": [],
+    "startup": ".\/Quake3-UrT-Ded.{{EXE}}  +exec \"server.cfg\" +set fs_game \"q3ut4\" +set fs_basepath \"\/home\/container\" +set fs_homepath \"\/home\/container\" +set dedicated \"2\" +set com_hunkmegs \"{{HUNKMEGS}}\" +set net_port \"{{SERVER_PORT}}\"",
+    "config": {
+        "files": "{\r\n    \"q3ut4\/server.cfg\": {\r\n        \"parser\": \"file\",\r\n        \"find\": {\r\n            \"set  sv_hostname\": \"set  sv_hostname \\\"{{server.build.env.SERVER_NAME}}\\\"\",\r\n            \"set  sv_joinmessage\": \"set  sv_joinmessage \\\"{{server.build.env.JOIN_MESSAGE}}\\\"\",\r\n            \"set  sv_maxclients\": \"set  sv_maxclients \\\"{{server.build.env.SERVER_MAXCLIENTS}}\\\"\",\r\n            \"set  rconpassword\": \"set  rconpassword \\\"{{server.build.env.RCON_PASSWORD}}\\\"\",\r\n            \"set  g_password\": \"set  g_password \\\"{{server.build.env.SERVER_PASSWORD}}\\\"\",\r\n            \"set  g_friendlyfire\": \"set  g_friendlyfire \\\"{{server.build.env.SERVER_FRIENDLYFIRE}}\\\"\",\r\n            \"set  g_gametype\": \"set  g_gametype \\\"{{server.build.env.SERVER_GAMETYPE}}\\\"\",\r\n            \"set  timelimit\": \"set  timelimit \\\"{{server.build.env.SERVER_TIMELIMIT}}\\\"\",\r\n            \"set  bot_enable\": \"set  bot_enable 1\",\r\n            \"set  bot_minplayers\": \"set  bot_minplayers \\\"{{server.build.env.BOT_MINPLAYERS}}\\\"\",\r\n            \"map  ut4_casa\": \"map  \\\"{{server.build.env.SERVER_MAP}}\\\"\"\r\n        }\r\n    }\r\n}",
+        "startup": "{\r\n    \"done\": \"------- Game Initialization -------\",\r\n    \"userInteraction\": []\r\n}",
```

Comment:
Userinteraction can be removed 0.7 leftover. Does not work anymore

Reply:
Removed Useraction.